subsys/bluetooth/controller/hci_driver.c

```diff
@@ -965,6 +965,17 @@ static int hci_driver_open(void)
 		return err;
 	}
 
```

Comment:
can you also set normal event length here using VS cmd and remove corresponding call to sdc_cfg_set(SDC_CFG_TYPE_EVENT_LENGTH)
That would be a nice unification / cleanup